package.json

```diff
@@ -57,7 +57,9 @@
   "homepage": "https://github.com/webex/react-ciscospark#readme",
   "dependencies": {
     "@ciscospark/common": "1.49.3",
+    "@ciscospark/common-timers": "1.49.3",
```

Comment:
Where are these used? Any idea why we need to specifically import them now?

Reply:
There were several tests failing if I didn't include them. Specifically one of the files was the built dist of `spark-core`. It complained it couldn't find the module `common-timers` for example.